src/cfml/system/modules_app/server-commands/commands/server/prune.cfc

```diff
@@ -0,0 +1,159 @@
+/**
+ * Prune.
+ * .
+ * {code:bash}
+ * server prune months=6
+ * server prune months=6 --list
+ * server prune months=6 --force
+ * {code}
+ **/
+component {
+
+	// di
+	property name="serverService" inject="ServerService";
+
+	/**
+	 * Prune.
+	 *
+	 * @months.hint forget servers which last started date is greater or equal to months you set
+     * @list.hint give a list of servers which last started date is greater or equal to the months you set
+	 **/    
+    function run(
+        string months,
+        Boolean list        = false,   
+        Boolean force       = false,
+    ){
+        var serverInfo      = serverService.resolveServerDetails( arguments ).serverinfo;
```

Comment:
What is the purpose of this line of code.  Not only is the contents of the `arguments` scope not useful as arguments to this method, you're not using the `serverInfo` variable below except for once which I believe is incorrect anyway.

Reply:
![image](https://user-images.githubusercontent.com/80276017/124826915-a1bfea00-df32-11eb-8638-ea918c57e1d3.png)

Reply:
that line is no more